targets/TARGET_NUVOTON/TARGET_M2354/TARGET_TFM/TARGET_NU_M2354/COMPONENT_TFM_S_FW/CMakeLists.txt

```diff
@@ -35,8 +36,9 @@ target_compile_definitions(mbed-m2354-tfm
     INTERFACE
         NU_TFM_S_BL2=$<IF:$<BOOL:${bl2}>,1,0>
         NU_TFM_S_MCUBOOT_IMAGE_NUMBER=${mcuboot_image_number}
-        # TODO: Fix escape sequence in NU_TFM_S_REGION_DEFS_H_PATH
+        # TODO: Fix escape sequence below
         #NU_TFM_S_REGION_DEFS_H_PATH=\"${region_defs_h_path}\"
+        #NU_TFM_S_PARTITION_MCU_H_PATH=\"${partition_mcu_h_path}\"
```

Comment:
Even if the headers are not used in most cases, it doesn't hurt to set them anyway? Or are we omitting them because of the escape sequence issue??

Reply:
They are used [here](https://github.com/OpenNuvoton/mbed/blob/a0757850f2f691fde74fbc94f1940ea1d27c69ba/targets/TARGET_NUVOTON/TARGET_M2354/device/nu_tfm_import_define.h#L43-L49). But as commented, they are hard-coded instead because the escape sequence issue is not fixed.

Reply:
Created #15032. It should fix the escaping of quotes, and those two line here can be uncommented.

Reply:
Actually the escape `\"` here doesn't make any difference compared to `"`, because CMake treats them as the same when storing this list.

Reply:
Add these two lines back and change to `"` from `\"`. Verify OK with #15032

Reply:
Thanks. Please rebase to master after #15032 is merged, and the PR check failure will also go away.

Reply:
Rebased